.gitpod.Dockerfile

```diff
@@ -1,4 +1,25 @@
-FROM gitpod/workspace-full
+FROM gitpod/workspace-base
+
+# Installing Node LTS
+# https://github.com/nvm-sh/nvm#installing-and-updating
+USER gitpod
+RUN curl -fsSL https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash \
+    && bash -c ". .nvm/nvm.sh \
+        && nvm install --lts \
+        && nvm alias default lts/* \
+        && npm install -g yarn pnpm"
```

Comment:
It's not used in Telescope, you can remove `yarn`.

Reply:
Thanks for the confirmation, removed